src/modules/ens/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const ENS_EMPTY_RESOLVER = '0x0000000000000000000000000000000000000000'
```

Comment:
I don't think this is inherently bad, but what do you think about having a constant that's not related to ENS and it's just "EMPTY_ADDRESS" ?
We could also replace `components/Modals/ContestModal/ContestModal.tsx`s placeholder on line 163.